dl/src/main/scala/com/intel/analytics/bigdl/nn/TimeDistributedCriterion.scala

```diff
@@ -29,77 +29,66 @@ import scala.reflect.ClassTag
  * @param critrn
  */
 
-class TimeDistributedCriterion[T : ClassTag](critrn : TensorCriterion[T])
+class TimeDistributedCriterion[T : ClassTag](critrn : TensorCriterion[T],
+                                             sizeAverage: Boolean = true)
```

Comment:
Yes. For recurrent nn, it is better to user no sizeAverage criterion training for quick converge. If the default is false, the convergence is slow. Please refer to the SequencerCriterion in torch rnn library, which set default value to false.